module_example/canvas_create_image.py

```diff
@@ -0,0 +1,13 @@
+from tkinter import *
```

Comment:
Is this class used?

Reply:
No this was just me trying things, I'll delete it